<?xml version="1.0" encoding="UTF-8"?>
<launch>

    <arg name="cloud_topic" default="/os_cloud_node/points" />

    <node pkg="lidar_camera_calibration" type="lidar_clustering" name="lidar_clustering" output="screen">
        <remap from="/point_cloud" to="$(arg cloud_topic)"/>
    </node>

    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find lidar_camera_calibration)/rviz/clustering.rviz" />

</launch>